api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -3406,4 +3408,33 @@ public void mergePatients_shouldNotFailIfMultiplePatientsHaveActiveOrderOfDiffer
 	public void processDeath_shouldThrowAPIExceptionIfPatientIsNull() throws Exception{
 		patientService.processDeath(null, new Date(), new Concept(), "unknown");
 	}
+
+	/**
+	 * @verifies move all the obs with same hierarchy for the patient after merge.
+	 * @see PatientService#mergePatients(org.openmrs.Patient, org.openmrs.Patient)
+	 */
+	@Test
+	@Verifies(value = "Should move all the obs with same hierarchy for the patient after merge.", method = "mergePatient(Patient)")
```

Comment:
@mddubey this is the old unit test style, see tests before yours. delete the verifies annotation above the method signature. not in the javadoc though.